<launch>

    <!-- PARAMETERS -->
    <param name="/use_sim_time" value = "true"/>

    <arg name = "bagfile" default = "/home/uib/bagfiles/colonia_sant_jordi/2020_11_03/11_17_42_2/12_25_27/topics_2020-11-03-11-17-53_0.bag"/>

    <!-- PLAY bagfile -->
    <node pkg = "rosbag" type = "play" name = "rosbag" args = "-r 1 --clock $(arg bagfile)" />


    <node pkg = "viso2_ros" type = "get_first_point" name = "get_first_point" output = "screen">
        <param name = "map_topic" value = "/turbot/navigator/ekf_map/odometry"/>
        <param name = "altitude_topic" value = "/turbot/navigator/altitude_raw"/>
        <param name = "outfile" value = "/home/uib/bagfiles/colonia_sant_jordi/2020_11_03/11_17_42_2/12_25_27/first_point.csv"/>
        <param name = "max_altitude" value = "2.5"/>
    </node>
<!-- 
    <node pkg = "viso2_ros" type = "stereo_odometer" name = "stereo_odometer" output = "screen" if = "$(eval enable_decimate_x1 == true)">
        <remap from = "stereo" to = "$(arg camera)"/>
        <remap from = "image" to = "image_rect"/>
        <remap from = "/turbot/navigator/altitude_raw" to = "altitude"/>
        <param name = "initial_translation_x" value = "$(arg t_x)"/>
        <param name = "initial_translation_y" value = "$(arg t_y)"/>
        <param name = "initial_translation_z" value = "$(arg t_z)"/>
        <param name = "initial_rotation_x" value = "$(arg q_x)"/>
        <param name = "initial_rotation_y" value = "$(arg q_y)"/>
        <param name = "initial_rotation_z" value = "$(arg q_z)"/>
        <param name = "initial_rotation_w" value = "$(arg q_w)"/>
    </node> -->

</launch>